app/controllers/admin/ticket_scannings_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class TicketScanningsController < Admin::BaseController
+    before_action :authenticate_user!
+    before_action :find_physical_ticket_by_token
+    load_and_authorize_resource through: :physical_ticket, singleton: true, shallow: true
```

Comment:
I am loading resource as well here though it is not needed for the create action but it is needed in defining it's abilities.